js/core/src/Blocks.svelte

```diff
@@ -290,12 +290,13 @@
 
 		function trigger_prediction(dep: Dependency, payload: Payload): void {
 			if (dep.trigger_mode === "once") {
-				if (!dep.pending_request) make_prediction(payload);
+				if (!dep.pending_request)
```

Comment:
Merging in main accidentally modified these lines

Reply:
I found these were revered in
https://github.com/gradio-app/gradio/pull/9318/files#diff-b1efd869b69b090780297c2060a3e07431340720a850ce6819598716c456ff4f.
@freddyaboulton Looks like other lines were also affected in the PR above.
Like https://github.com/gradio-app/gradio/pull/9318/files#diff-b1efd869b69b090780297c2060a3e07431340720a850ce6819598716c456ff4f and 
https://github.com/gradio-app/gradio/pull/9318/files#diff-b1efd869b69b090780297c2060a3e07431340720a850ce6819598716c456ff4fR287
Can you check if it's ok, while they are not directly related to this PR.

Reply:
Thank you! I think it's looking good now